conf/ocsci/submariner_downstream.yaml

```diff
@@ -0,0 +1,7 @@
+ENV_DATA:
+  submariner_source: "downstream"
+  submariner_url: "https://get.submariner.io"
```

Comment:
Is this used also in downstream deployment? I thought that this is upstream page, so curious what it's doing in DS deployment config.